tests/trust/test_trust.yml

```diff
@@ -48,4 +52,12 @@
       register: check_add_trust
       failed_when: "'1 trust matched' not in check_add_trust.stdout"
 
-    when: trust_test_is_supported | default(false)
\ No newline at end of file
+    - name: delete trust
```

Comment:
This is unexpected with the commit title.

Reply:
fixed.